dhis-2/dhis-services/dhis-service-reporting/src/test/java/org/hisp/dhis/interpretation/InterpretationServiceTest.java

```diff
@@ -55,10 +57,11 @@
  * @author Lars Helge Overland
  */
 public class InterpretationServiceTest
-    extends DhisSpringTest
+    extends
+    DhisSpringTest
 {
     @Autowired
-    private UserService userService;
+    private UserService _userService;
```

Comment:
Can you just use "userService" here?

Reply:
I don't think so. We need to inject userService in the test otherwise you get an error `java.lang.IllegalArgumentException: UserService must be injected in test`. If you look at `setUpTest` you will see that we are doing:
```
    @Override
    protected void setUpTest() throws Exception
    {
        userService = _userService;
    }
```

This is widely used in a lot of test in Dhis2. Look for `userService = _userService` and you will find more than 80 occurrences.

Reply:
OK thanks.